gallery/src/components/NxGlobalSidebar/NxGlobalSidebarExample.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxGlobalSidebar, NxGlobalSidebarNavigation, NxGlobalSidebarNavigationLink, useToggle, NxP, NxH3, NxPageMain }
+  from '@sonatype/react-shared-components';
+import { faArrowLeft, faArrowRight, faLink, faExternalLinkSquareAlt } from '@fortawesome/free-solid-svg-icons';
+
+export default function NxGlobalSidebarExample() {
+  const [sidebarOpen, onToggleCollapse] = useToggle(true);
+  return (
+    <div className="nx-page-content">
+      <NxGlobalSidebar isOpen={sidebarOpen}
+                       toggleOpenIcon={faArrowLeft}
+                       toggleCloseIcon={faArrowRight}
+                       onToggleClick={onToggleCollapse}
+                       logoImg="./assets/images/logo-plaid-villain-text.png"
```

Comment:
This should be pulled in via a `require` call

Reply:
> This should be pulled in via a `require` call

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/10799b9adea5f60d3454929b3771140cc3d62dd8